### DISCUSSION TITLE
Adding sprites to protomaps similar to the protomaps example

### DISCUSSION THREAD

**Discussion by Idan-Garay** [protomaps/basemaps#266] [Q&A]
Hi, I need help with adding sprites to https://api.protomaps.com/tiles/v3.json?key=aed0160c33ad7d26. I'd like to follow this https://raw.githubusercontent.com/protomaps/.github/main/profile/dark.jpg protomap where icon-images can be seen using protomaps as a source.

I've attempted to use OSM Liberty in Maputnik and replace the TileJSON URL from maptiler to protomaps which resulted to a map with no details. Can anyone recommend a link that I can refer when adding a spritesheet to a JSON map?

**Comment by bdon** [comment]
The icons you see in the screenshot haven't been implemented in this repository yet.

**Reply by Idan-Garay** [reply]
Thank you for answering. How much effort is required to add sprites to a Protomaps basemap? Can we use existing spritesheets from other tile providers?

**Reply by bdon** [reply]
Other spritesheets will likely not* work because they don't specify the same icon names, and may not have a compatible license.

**Reply by Idan-Garay** [reply]
I see, I was able to add sprites. The problem lies with the style layers for POIs with sprites, I think adding POIs similar to other existing tile providers like Mapbox isn't feasible and isn't as simple as copy and pasting the style layers and extending it to protomaps's tile source. Also, I think adapting the other tile providers' style layers to protomaps might be very labor-intensive

**Reply by bdon** [reply]
Sorry, I realize I wrote the wrong thing above - I meant to write "not work", which you pointed out is the case. 

For a provider like Mapbox their POI artwork and client software is not open source so it is not possible to adapt because of the license.
